include/acvp/acvp_lcl.h

```diff
@@ -13,7 +13,7 @@
 #include "parson.h"
 
 #define ACVP_VERSION    "1.0"
-#define ACVP_LIBRARY_VERSION    "libacvp_oss-1.0.0"
+#define ACVP_LIBRARY_VERSION    "libacvp_oss-1.0.1"
```

Comment:
May want to update micro version in configure.ac as well, though that can be done in a later PR if we are in a hurry

Reply:
I'll update it now